pyramda/range.py

```diff
@@ -0,0 +1,8 @@
+from pyramda.function.curry import curry
+
+arange = range
```

Comment:
You can `import builtins` and refer to range as `builtins.range` to avoid this renaming.